### DISCUSSION TITLE
Could you provide an example of MFE host that resolves apps entries in runtime?

### DISCUSSION THREAD

**Discussion by gabrieldalborgo** [web-infra-dev/modern.js#5058] [Q&A] [answered]
I found this section in the documentation ([https://modernjs.dev/en/guides/topic-detail/micro-frontend/c03-main-app.html#custom-remote-app-list](https://modernjs.dev/en/guides/topic-detail/micro-frontend/c03-main-app.html#custom-remote-app-list)) regarding resolving the app list at runtime, but I can't make it work. "getAppList" function is never executed

Do you have a simple sample of how it works?

**Comment by kyrylobashtenko** [comment, marked as answer]
@gabrieldalborgo Check out this repository: https://github.com/kirillbashtenko/modern-js-example.git. Each of the three projects needs to be run separately, as I haven't implemented a unified command for simultaneous execution. However, they should work smoothly, and getAppList executes correctly upon each load.

**Comment by gabrieldalborgo** [comment]
Thanks for the sample. It's all I needed.
